src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,31 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $_REQUEST['s'] ) ) {
+			// nonce check if search form submitted.
+			$nonce = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? $_POST['aggregator']['nonce'] :  '';
+			if ( $nonce === '' || wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
```

Comment:
I think the first conditional is not required.

```suggestion
			if ( wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
```